pkg/plugin/framework/backup_item_action.go

```diff
@@ -31,13 +32,15 @@ type BackupItemActionPlugin struct {
 	*pluginBase
 }
 
+var _ plugin.GRPCPlugin = &BackupItemActionPlugin{}
```

Comment:
1- what does this do?
2- how did you figure out this was needed?

Reply:
This ensures we get a compile-time error if `*BackupItemActionPlugin` doesn't implement the `plugin.GRPCPlugin` interface.  It's declaring a package-level variable, with no name (`var _`, of type `plugin.GRPCPlugin`, and assigning to it a `&BackupItemActionPlugin{}` instance.  If `&BackupItemActionPlugin{}` doesn't implement `plugin.GRPCPlugin`, then this line won't compile.

Reply:
Ah. Is there anyway we could do this in a test instead, to avoid having a pkg level variable?